src/main/java/com/ruralsourcing/WeatherBot/service/SlackIntegration.java

```diff
@@ -0,0 +1,99 @@
+package com.ruralsourcing.WeatherBot.service;
+
+import com.github.seratch.jslack.Slack;
+import com.github.seratch.jslack.api.model.event.MessageEvent;
+import com.github.seratch.jslack.api.rtm.RTMClient;
+import com.github.seratch.jslack.api.rtm.message.Message;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.ruralsourcing.WeatherBot.model.ResponseModel;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+
+/**
+ *
+ */
+public class SlackIntegration implements Runnable {
+    private static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+    private static final String TOKEN = System.getenv("SLACK_BOT_API_TOKEN");
+
+    private final AtomicReference<ResponseModel> reference;
+
+    public SlackIntegration(AtomicReference<ResponseModel> reference) {
+        this.reference = reference;
+    }
+
+    @Override
+    public void run() {
+        rtmMessage();
+    }
+
+    private void rtmMessage() {
+        final Slack slack = Slack.getInstance();
+
+        try (final RTMClient rtm = slack.rtm(TOKEN)) {
+            rtm.addMessageHandler((message) -> {
```

Comment:
Parenthesis around the parameter in a single-parameter lambda can be omitted: `rtm.addMessageHandler(message -> {`.